controllers/admin/AdminThemesController.php

```diff
@@ -142,16 +142,24 @@ public function initPageHeaderToolbar()
         parent::initPageHeaderToolbar();
 
         if (empty($this->display)) {
-            $this->page_header_toolbar_btn['import_theme'] = array(
-                'href' => self::$currentIndex.'&action=importtheme&token='.$this->token,
-                'desc' => $this->trans('Add new theme', array(), 'Admin.Design.Feature'),
-                'icon' => 'process-icon-new'
-            );
-            $this->page_header_toolbar_btn['export_theme'] = array(
-                'href' => self::$currentIndex.'&action=exporttheme&token='.$this->token,
-                'desc' => $this->trans('Export current theme', array(), 'Admin.Design.Feature'),
-                'icon' => 'process-icon-export'
-            );
+            if (!in_array(
+                $this->authorizationLevel(),
+                array(
+                    AdminController::LEVEL_VIEW,
+                    AdminController::LEVEL_EDIT,
```

Comment:
No it's normal.
Here you check the rights to display the button, below you check the rights to display an error message, so it's normal to check the opposite permissions.